### PR TITLE
chore: refactor issue creator multiplier

### PR DESCRIPTION
**Pull request by gitcoindev**
Resolves: #583

Follow up to https://github.com/ubiquity/ubiquibot/pull/588 which was closed and cannot be reopened due to force push.

QA done, comments history is available at #588.

### CODE REVIEW

**Comment by web4er** [comment]
> QA done, comments history is available at https://github.com/ubiquity/ubiquibot/pull/588.

I didn't understand it. What do you mean by this?

File: .github/ubiquibot-config.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 evm-network-id: 100
 price-multiplier: 1.5
+issue-creator-multiplier: 1
```
Comment (web4er) [inline .github/ubiquibot-config.yml:3]: This should be 2 as given in the issue discussion. https://github.com/ubiquity/ubiquibot/issues/583#issue-1838350837

**Review by web4er — CHANGES_REQUESTED** [review]

**Comment by gitcoindev** [comment]
> > QA done, comments history is available at #588.
> 
> I didn't understand it. What do you mean by this?

Hi @web4er thank you for starting the review, I meant QA linked to the original pr 
image

File: .github/ubiquibot-config.yml (hunk 1 above)
Comment (gitcoindev) [reply .github/ubiquibot-config.yml]: done

**Comment by web4er** [comment]
> Hi @web4er thank you for starting the review, I meant QA linked to the original pr

The QA that you have to provide is showing the changes working using your local instance or QA bot instance  by creating an issue and explaining step by step. https://github.com/ubiquity/ubiquibot#how-to-qa-any-additions-to-the-bot

QA done by workflow or the team is different.

**Comment by gitcoindev** [comment]
@web4er I currently cannot execute QA due to https://github.com/ubiquity/ubiquibot/issues/620 , and would be grateful if the project team did the QA and verified the PR, as discussed here https://github.com/ubiquity/ubiquibot/pull/588#issuecomment-1673202715 .

**Comment by gitcoindev** [comment]
Hi @web4er , @pavlovcik according to https://github.com/orgs/ubiquity/projects/23/views/1 the QA status column is done for #588 , does it mean that this pull request can be merged?

**Review by pavlovcik — DISMISSED** [review]
Code looks good lets get this tested!

**Comment by gitcoindev** [comment]
Hi @web4er , the bot is pinging me about lack of progress. I would be grateful if you tested and provided feedback.

**Comment by pavlovcik** [comment]
Also @Draeieg when you get the chance

**Comment by whilefoo** [comment]
QA: https://github.com/ubiquity-whilefoo/ubiquibot/issues/27

Error I get in terminal:
```
ERROR (event): require() of ES Module ubiquity/ubiquibot/node_modules/mdast-util-from-markdown/index.js from ubiquity/ubiquibot/lib/src/helpers/comment.js not supported.
Instead change the require of index.js in ubiquity/ubiquibot/lib/src/helpers/comment.js to a dynamic import() which is available in all CommonJS modules.
```

@gitcoindev can you please merge latest development branch into this PR because there were changes related to issue creator reward

**Comment by gitcoindev** [comment]
Hi @whilefoo thank you for checking. Normally I would do a rebase on top of latest development but this would require force push which I tried already here, will do the merge now and resolve conflict as you suggested.

**Comment by gitcoindev** [comment]
@whilefoo all good, I merged latest development branch and resolved conflicts, please check now.

**Comment by pavlovcik** [comment]
> @gitcoindev can you please merge latest development branch into this PR because there were changes related to issue creator reward

There is a button on the GitHub pull request view UI (here) at the bottom that allows you to get the latest changes from `development`. I believe it just performs a simple merge on behalf of the bounty hunter. I would encourage direct changes like this instead of waiting for the bounty hunter to do it.

**Comment by web4er** [comment]
> The branch is out-of-date again after new updates, but I believe @pavlovcik 's approach can be used after the requested change is approved by @web4er .

I am going to do a QA today and then will approve it.

**Review by web4er — APPROVED** [review]
Working well. 

QA(part-1): https://github.com/web4erOrg/ubiquibot/issues/72
QA(part-2): https://github.com/web4erOrg/ubiquibot/issues/73
QA(part-3): https://github.com/web4erOrg/ubiquibot/issues/74

**Comment by gitcoindev** [comment]
Hi @pavlovcik it seems that `Preview deployment` check is missing because it expired and it blocks merging, can this be triggered again?

**Comment by pavlovcik** [comment]
> Hi @pavlovcik it seems that `Preview deployment` check is missing because it expired and it blocks merging, can this be triggered again?

I just deleted the deployments on the repository settings. It didn't do anything but hopefully I didn't break something else.

**Comment by gitcoindev** [comment]
Thank you for merging!

**Pull request merged**